lxd/network/driver_common.go

```diff
@@ -749,9 +756,15 @@ func (n *common) bgpGetPeers(config map[string]string) []string {
 		peerAddress := config[fmt.Sprintf("bgp.peers.%s.address", peerName)]
 		peerASN := config[fmt.Sprintf("bgp.peers.%s.asn", peerName)]
 		peerPassword := config[fmt.Sprintf("bgp.peers.%s.password", peerName)]
+		peerHoldTime := config[fmt.Sprintf("bgp.peers.%s.holdtime", peerName)]
+
+		// If hold time not configured use default 3min value
```

Comment:
full stop "." at end of sentences please.

Reply:
Done.
What do you think about this part:
peerHoldTime = "180"
Maybe we need to create something like this
`const DEFAULT_BGP_HOLDTIME = "180"`
and modify
`peerHoldTime = DEFAULT_BGP_HOLDTIME `
We can do such thing with all other variables like `MultihopTtl: 255` in the future.

Reply:
Thanks, no need for constants right now as the default is only defined once in code.